Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -419,6 +429,16 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("GetOutlineSize")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddExpression(
+        "OutlineSize",
+        _("Get current outline size"),
```

Comment:
Just name this expression "Outline size".

When you're browsing expressions, you now that you'll "get the current" value. So no need for "Get current ...". Just state what the expression is returning: "Outline size"